### PR TITLE
Add Session Feedback for speakers

### PR DESCRIPTION
**Pull request by froboy**
This creates a new view (that's not super well styled, but it should work for now) to display session feedback on session nodes. The block should only be visible to admins and users listed as speakers on the session.

To test:
- visit http://nginx-midcamp-org-feature-session-feedback.us.amazee.io/2020/topic-proposal/build-it-better-stupid-using-design-systems-scalability as anon and observe there's nothing between the Captions and the footer.
- Submit some feedback.
- login as admin and see the block
- login as a speaker on the node: `drush uli 1366` and observe the block is still there
- login as another user: `drush uli 123` and observe the block is hidden.

To do:
- Getting results to display required giving all Authenticated users "View all webform results" permission. This might... not be optimal, but I didn't get to test the implications.

### CODE REVIEW

**Comment by froboy** [comment]
This issue seems to be a good description of the permission issue I ran into: https://www.drupal.org/project/webform/issues/2965814. I'm looking into a custom solution at the moment, but... up for discussion at to whether or not this is good to ship.

**Comment by Brian-Clement** [comment]
Thanks for this @froboy .  I'm going to think on this a bit, right now when I'm logged in as any random user I can see the full result set at http://midcamp.org.docker.amazee.io/admin/structure/webform/manage/session_feedback/results/submissions.  The same goes for all the other webforms on the site.

Probably not the end of the world, as we're not really capturing sensitive info here, but it still feels bad.

**Comment by froboy** [comment]
Yea, agreed. I was thinking about creating a new permission called
something like “view my session feedback” and using a similar logic on it.
I’ll try and hack on it over the weekend.

File: conf/drupal/config/webform.webform.session_feedback.yml [hunk 1]
```diff
@@ -152,7 +152,8 @@ access:
     users: {  }
     permissions: {  }
   view_own:
-    roles: {  }
+    roles:
+      - authenticated
```
Comment (froboy) [inline conf/drupal/config/webform.webform.session_feedback.yml:156]: @Brian-Clement this is unrelated to the submissions block, right? It's just so users can see what they submitted?

Comment (Brian-Clement) [reply conf/drupal/config/webform.webform.session_feedback.yml:156]: Sorry @froboy, hadn't gotten around to writing up some comments here (sorry for jumping in and pushing to your branch like this). 

The crux of what I wanted to do was to leverage `hook_webform_submission_access()` to conditionally grant users permission to view results (based on the logic you introduced in `hook_block_access()`).  The strange thing is that `hook_webform_submission_access()` was never firing at all until I added some some `view` permissions to this particular form.

Once I added this I was able to start seeing `hook_webform_submission_access()` at play so I could start adding logic there.  The disabling of SQL rewrite in the View was also done for this same purpose--sort of a workaround that lead to being able to bypass some of the inner workings of webform's permissions.

**Comment by froboy** [comment]
@Brian-Clement thanks for this, it looks great. I've tested and it functions as expected, without the... badness from before.

File: web/modules/custom/midcamp_utility/midcamp_utility.module [hunk 2]
```diff
@@ -100,3 +105,75 @@ function midcamp_utility_entity_view_mode_alter(&$view_mode, Drupal\Core\Entity\
...
+/**
+ * Implements hook_webform_submission_access().
+ */
+function midcamp_utility_webform_submission_access(WebformSubmissionInterface $webform_submission, $operation, AccountInterface $account) {
+  // Do not adjust access for any other webforms.
+  if ($webform_submission->getWebform()->id() != 'session_feedback') {
+    return AccessResult::neutral();
+  }
+
+  // Determine what type of page we are on.
```
Comment (froboy) [inline web/modules/custom/midcamp_utility/midcamp_utility.module:155]: It'd be good to dedupe this section (155-176) into a helper function, but that can be for later.
